pkg/trtl/config/config.go

```diff
@@ -131,6 +144,24 @@ func (c *ReplicaConfig) Validate() error {
 	return nil
 }
 
+// Strategies extracts the replica configuration strategies from the configuration
+func (c *ReplicaStrategyConfig) Strategies() (strategies []ReplicaStrategy) {
+	strategies = make([]ReplicaStrategy, 0)
+	if c.HostnamePID {
+		strategies = append(strategies, HostnamePID(c.Hostname))
+	}
+
+	if c.FilePID != "" {
+		strategies = append(strategies, FilePID(c.FilePID))
+	}
+
+	if c.JSONConfig != "" {
+		strategies = append(strategies, JSONConfig(c.JSONConfig))
+	}
+
+	return strategies
+}
```

Comment:
Note the fixed ordering of strategies - it's not very flexible, but punting this for the future.